<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagSelectDialog</class>
 <widget class="QDialog" name="tagSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Available Tags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>New Tag</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="newTagEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="createTagButton">
          <property name="text">
           <string>Create Tag</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="mainTagList">
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <column>
         <property name="text">
          <string>Tags</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="addTagToListButton">
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doneButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
